### PR TITLE
PFM-TASK-4114 - Adjust cplace-asc to use newest Node 18

### PR DESCRIPTION
**Pull request by slaven3kopic**
Resolves [PFM-TASK-4114](https://base.cplace.io/pages/axc0htmcfrwkqxy6ch6ujbpfz/PFM-TASK-4114-Adjust-cplace-asc-to-use-newest-Node-18?highlightedEntityUid=page%2Faxc0htmcfrwkqxy6ch6ujbpfz)

**Checklist:**
- [x] Version updated (if applicable)
- [ ] Tests added (if applicable)
- [ ] Code formatted
- [x] Chosen correct branch as a merge target (For @cplace/asc use the legacy-asc branch, for @cplace/asc-local use master, or a release branch like 2.x.x etc.)
- [x] Milestone added
- [x] PR labels added

### CODE REVIEW

**Review by vladimir-arsov — COMMENTED** [review]
In the new skeleton we changed the node version to 18.18.0. Should we update it there to 18.18.2?
Also should be update the package.json of the skeleton to point to cplace-asc 2.1.0?

**Comment by slaven3kopic** [comment]
> In the new skeleton we changed the node version to 18.18.0. Should we update it there to 18.18.2? Also should we update the package.json of the skeleton to point to cplace-asc 2.1.0?

Yes, for both. Thanks

**Review by vladimir-arsov — APPROVED** [review]

**Review by vladimir-arsov — COMMENTED** [review]
One thing I just thought of. Don't you need a different branch for 2.1 releases? Otherwise how would you publish a new 2.0 release which should still be maintained?

**Comment by slaven3kopic** [comment]
> One thing I just thought of. Don't you need a different branch for 2.1 releases? Otherwise how would you publish a new 2.0 release which should still be maintained?

Good point @vladimir-arsov . We will create branch `2.0.x` and `2.1.x.` `2.x.x `will not be needed afterwards

File: tools/asc-local-migration/asc-2-asc-local.ts [hunk 1]
```diff
@@ -15,8 +15,8 @@ const NODE_MODULES_IGNORE_LINE = 'node_modules/';
 const CPLACE_ASC_LOCAL_PKG_NAME = '@cplace/asc-local';
 const CPLACE_ASC_LOCAL_PKG_VERSION = '^2.0.7';
 const PACKAGES_TO_REMOVE = ['typescript'];
-const NODE_VERSION = '18.11.0';
-const NPM_VERSION = '8.19.2';
+const NODE_VERSION = '18.18.2';
```
Comment (JanDittmar) [inline tools/asc-local-migration/asc-2-asc-local.ts:18]: Can't we use `SupportedNodeVersions.SUPPORTED_NODE_VERSION` for that? That way we'd have one place less to change next time. 👀

**Review by JanDittmar — APPROVED** [review]
Looking good.

**Review by Kaltenbach — APPROVED** [review]
LGTM

**Pull request merged**